src/helix/endpoints/moderation/mod.rs

```diff
@@ -74,6 +110,12 @@ pub use update_automod_settings::{
 #[cfg(feature = "beta")]
 #[doc(inline)]
 pub use warn_chat_user::{WarnChatUser, WarnChatUserBody, WarnChatUserRequest};
+// endpoints defined in other modules.
+pub use super::channels::add_channel_vip::{self, AddChannelVipRequest, AddChannelVipResponse};
+pub use super::channels::get_vips::{self, GetVipsRequest, Vip};
+pub use super::channels::remove_channel_vip::{
+    self, RemoveChannelVipRequest, RemoveChannelVipResponse,
+};
```

Comment:
added this to make it more inline with what twitch thinks the category should be.

The reason it is in channels is because the endpoints are on `helix/channels/*`